modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItTablesApiTest.java

```diff
@@ -116,97 +125,314 @@
 
     /** Cluster nodes. */
     private List<Ignite> clusterNodes;
-
+    
     /**
      * Before each.
      */
     @BeforeEach
     void beforeEach(TestInfo testInfo) throws Exception {
         String metastorageNodeName = IgniteTestUtils.testNodeName(testInfo, 0);
-
+        
         clusterNodes = IntStream.range(0, nodesBootstrapCfg.size()).mapToObj(value -> {
                     String nodeName = IgniteTestUtils.testNodeName(testInfo, value);
-
+                    
                     return IgnitionManager.start(
                             nodeName,
                             nodesBootstrapCfg.get(value).apply(metastorageNodeName),
                             workDir.resolve(nodeName));
                 }
         ).collect(Collectors.toList());
     }
-
+    
     /**
      * After each.
      */
     @AfterEach
     void afterEach() throws Exception {
         IgniteUtils.closeAll(ItUtils.reverse(clusterNodes));
     }
+    
+    /**
+     * Trys to create a table which is already created.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testTableAlreadyCreated() throws Exception {
+        clusterNodes.forEach(ign -> assertNull(ign.tables().table(TABLE_NAME)));
+        
+        Ignite ignite0 = clusterNodes.get(0);
+        
+        Table tbl = createTable(ignite0, SCHEMA, SHORT_TABLE_NAME);
+        
+        assertThrows(TableAlreadyExistsException.class,
```

Comment:
we need to check both nodes are throw such exceptions also client nodes too.

Reply:
I am afraid if I check for each node, the test would be flaky until IGNITE-15891 is fixed.

Reply:
let it be the test with @Ignored and appropriate todo ?

Reply:
I already wrote such tests.
I want to see even one which will passed well.